checks.d/zk.py

```diff
@@ -96,6 +96,20 @@ def check(self, instance):
                 self.service_check('zookeeper.mode', status, message=message,
                                    tags=sc_tags)
 
+        # Read ouput from `mntr` output.
+        try:
+            mntr_out = self._send_command('mntr', *cx_args)
+        except ZKConnectionFailure:
+            self.increment('zookeeper.timeouts')
```

Comment:
@fw42 I'm not sure if it's trying to report zookeeper timeouts or datadog failure timeouts.
Just made me think though, we don't want to be duplicating the increment when both `stat` and mntr` fail because that will report 2 failures?